README.md

```diff
@@ -43,10 +64,26 @@ Datalake is a single storage that stores different types of monitoring data such
 as logs, metrics, traces, events. The single storage allows for far more
 corelation than different data being stored at different places.
 
+### DevOps
+
+DevOps is a set of practices that combines software development and operations to shorten the development life cycle and improve reliability. The idea behind the DevOps movement is to reduce silos between different teams and make software delivery faster and more reliable via automation and collaboration.
+
+### Downsampling
+
+Downsampling is the process of reducing the amount of data points in a time series to make it more manageable without losing the essence and can help reduce the cost of storage and performance of querying. 
+
 ## E
 
+### Event
+
+An event is a significant occurrence within a system that may require attention. Events can be used to trigger alerts or notify operators of potential issues.
```

Comment:
```suggestion
Events are hard to define because everything is an event, but let's give it a try 😉. An event is primarily a change event that can mean a pod restart, deployment, or configuration flag change. Events are important because they affect the system's state externally and can help correlate incidents with metrics, traces, and logs.
```